gravitee-apim-cypress/cypress/model/apis.ts

```diff
@@ -15,7 +15,7 @@
  */
 export interface Api {
   id: string;
-  contextPath: string;
+  context_path: string;
```

Comment:
doing this you break `quality.spec.ts`, `visibility.spec.ts`, `visibility.spec.ts`
please update them

Reply:
Ok, will revert that change.